lib/utils.js

```diff
@@ -3,7 +3,11 @@ export function getGlobalScope() {
 }
 
 export function isWebEnvironment()  {
-  return getGlobalScope() !== null;
+  return typeof document != 'undefined';
```

Comment:
Should be `!==` or intentionally using `!=`?

Reply:
Typo, I update that now.